assets/replies.html

```diff
@@ -2,14 +2,14 @@
 @@-- $2 is how to call a single reply
 @@-- $3 is how to call multiple replies
 
-<div class="replies">
-    <div class="replies-total">
+<section class="replies">
+    <h3 class="replies-total">
```

Comment:
This changes text properties (makes the text bold at the very least, seemingly also adds a margin around it), I'd rather it not be an h3.